src/main/java/org/broadinstitute/hellbender/dev/pipelines/bqsr/BaseRecalibratorDataflowUtils.java

```diff
@@ -90,6 +107,33 @@ public static void ensureReferenceIsReadable(final PipelineOptions popts, String
   // ---------------------------------------------------------------------------------------------------
   // non-public methods
 
+
+  private static PCollection<BaseRecalOutput> addQuantizationInfo(SAMFileHeader readsHeader, BaseRecalibrationArgumentCollection toolArgs, PCollection<RecalibrationTables> recal) {
+    return recal.apply(ParDo
+            .named("addQuantizationInfo")
+            .of(new DoFn<RecalibrationTables, BaseRecalOutput>() {
+              @Override
+              public void processElement(ProcessContext c) {
+                RecalibrationTables rt = c.element();
+                BaseRecalibratorWorker baseRecalibratorWorker = BaseRecalibratorWorker.fromArgs(readsHeader, toolArgs);
+                baseRecalibratorWorker.onTraversalStart(null);
+                //BaseRecalOutput ret = new BaseRecalOutput(rt, baseRecalibratorWorker.getQuantizationInfo(rt), baseRecalibratorWorker.getRequestedCovariates());
+
+                // Saving and loading back the report actually changes it. So we have to do it.
+                // TODO: Figure out what it changes, and just do that instead of doing the whole rigamarole.
+                try {
+                    File temp = BaseTest.createTempFile("temp-recalibrationtable-",".tmp");
+                    toolArgs.RAC.RECAL_TABLE = new PrintStream(temp);
```

Comment:
known sites is a custom type that keeps tabs on a file. That doesn't translate well when we're on a different machine that doesn't have this file. Or when the knownSites are on GCS instead of a File.
In this new world, the un-scary thing to do is to explicitly ask to read knownSites when and where we are ready to do it. In this code (since I was told "wait for skeleton it'll have code to read variants from GCS"), the right time to do it is on the client, and then we put those into a PCollection. knownSites are only there for command-line parsing.